scene/main/viewport.cpp

```diff
@@ -1314,7 +1314,7 @@ bool Viewport::_gui_call_input(Control *p_control, const Ref<InputEvent> &p_inpu
 	Ref<InputEvent> ev = p_input;
 
 	// Returns true if an event should be impacted by a control's mouse filter.
-	bool is_mouse_event = Ref<InputEventMouse>(p_input).is_valid();
+	bool is_mouse_event = Ref<InputEventMouse>(p_input).is_valid() || Ref<InputEventScreenDrag>(p_input).is_valid() || Ref<InputEventScreenTouch>(p_input).is_valid();
```

Comment:
Let me suggest either renaming the variable (e.g., `is_pointer_event`) or adding a new one and checking both.

Reply:
Since the variable is only used in a single place, splitting it up into two variables seems like a bit of overkill.